### PR TITLE
Assignment 1

### PR DESCRIPTION
**Pull request by l-mack**
## Checklist
- [ ] I can confirm that my changes are working as intended

### CODE REVIEW

File: Copy_of_lab_1.ipynb
Comment (tianyi21) [inline Copy_of_lab_1.ipynb:1]: * Feed Forward Neural Networks with Keras:
  *  Please always reinitialize your model with `model = Sequential() ...` to reset all model parameters. Or, you are continuing the training from the previous trained state with a different model setup. You may check this with the accuracy of the first epoch.
  * Please train the model with the extra hidden layer.

File: 01_materials/labs/lab_2.ipynb
Comment (tianyi21) [inline 01_materials/labs/lab_2.ipynb:1]: - b: The output probabilities from `plot_prediction` doesn't look right. Based on the training curve, the model should make better predictions compared with the un-trained model we saw above.
- c: Please find the plot the worst sample with `numpy`.
- Hyper parameters settings: it seems that the accuracy was ~10% in the end. Is this the best test accuracy you can achieve?

File: 02_activities/assignments/assignment_1.ipynb
Comment (tianyi21) [inline 02_activities/assignments/assignment_1.ipynb:1]: - Task 1: You may utilize the provided class_name as class_names[y_train[...]] to get readable class names.

**Review by tianyi21 — APPROVED** [review]
Hi Laura!

Thank you for submitting your DL assignments. Please refer to my comments attached to each file.

Your A1 and Lab 1-3 are now complete.

Thanks,
Tianyi [LS]
